test/e2e/apps/disruption.go

```diff
@@ -491,7 +491,7 @@ func waitForPodsOrDie(cs kubernetes.Interface, ns string, n int) {
 		ready := 0
 		for i := range pods.Items {
 			pod := pods.Items[i]
-			if podutil.IsPodReady(&pod) {
+			if podutil.IsPodReady(&pod) && pod.ObjectMeta.DeletionTimestamp.IsZero() {
```

Comment:
`DeletionTimestamp` is a pointer so this needs to be `pod.ObjectMeta.DeletionTimestamp != nil`

Reply:
sent a PR to fix it https://github.com/kubernetes/kubernetes/pull/93501